src/Runner.Common/JobServerQueue.cs

```diff
@@ -476,8 +471,21 @@ private async Task ProcessSummaryUploadQueueAsync(bool runOnce = false)
                         }
                         catch (Exception ex)
                         {
+                            var issue = new Issue() { Type = IssueType.Warning, Message = $"Caught exception during summary file upload to results. {ex.Message}" };
+                            issue.Data[Constants.Runner.InternalTelemetryIssueDataKey] = Constants.Runner.ResultsUploadFailure;
+
+                            var telemetryRecord = new TimelineRecord()
+                            {
+                                Id = Constants.Runner.TelemetryRecordId,
+                            };
+                            telemetryRecord.Issues.Add(issue);
```

Comment:
```suggestion
                            var telemetryRecord = new TimelineRecord()
                            {
                                Id = Constants.Runner.TelemetryRecordId,
                            };
                            telemetryRecord.Issues.Add(new Issue() { Type = IssueType.Warning, Message = $"Caught exception during summary file upload to results. {ex.Message}" });
```

Reply:
I still want `data` field so I can trace and filter on it.  If we use this to pass up telemetry in the future, I don't want to see other failiures.  

Otherwise I have to rely on a sentence, I would much prefer to filter on a key instead.